core/arch/arm/mm/tee_mmu.c

```diff
@@ -648,6 +648,27 @@ TEE_Result vm_get_flags(struct user_mode_ctx *uctx, vaddr_t va, size_t len,
 	return TEE_SUCCESS;
 }
 
+TEE_Result vm_get_attrs(struct user_mode_ctx *uctx, vaddr_t va, size_t len,
```

Comment:
I'd prefer `vm_get_prot()` and mask out only the `prot` bits from `r->attr` below.
This function should go into a separate commit also.